### PR TITLE
adding e8 scan policy

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/7479

### CODE REVIEW

File: security/create_e8_scan_pol.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+. Log in to your hub cluster.
+
+. From the navigation menu, select *Govern risk*. 
+
+. Click *Create policy*. As you complete the YAML form, select *E8Scan* from the _Specifications_ field.
++ 
+The following resources are created: ScanSettingBinding, a ComplianceSuite (`compliance-suite-e8`), and a ComplianceCheckResult (`compliance-suite-e8-results`).
++
+*Note:* Enforce is supported, however it is best to set the remediation action to `inform`.
```
Comment (ycao56) [inline security/create_e8_scan_pol.adoc]: The policy has to be enforced in order to create scansettingbinding

File: security/e8_scan_policy.adoc [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+[#e8-scan-policy-resources]
+== E8 scan policy resources
+
+When you create an E8 scan policy the following resources are created:
+
+- A ScanSettingBinding resource to identify which profiles to scan:
+
++
+----
+apiVersion: compliance.openshift.io/v1alpha1
```
Comment (ycao56) [inline security/e8_scan_policy.adoc]: can you wrap this CR with configuration policy? as below two yamls have configpolicy wrapper

Comment (dockerymick) [reply security/e8_scan_policy.adoc]: done

File: security/create_e8_scan_pol.adoc (hunk 1 above)
Comment (dockerymick) [reply security/create_e8_scan_pol.adoc]: Updated

**Review by ycao56 — APPROVED** [review]

**Comment by dockerymick** [comment]
@ycao56 Thanks for your review! @ycao56

**Pull request merged**